frontend/Exence/src/app/shared/display-theme.service.ts

```diff
@@ -0,0 +1,86 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, map, Observable } from 'rxjs';
+
+export enum DisplayTheme {
+	DARK = 'DARK',
+	LIGHT = 'LIGHT'
+}
+
+interface ThemeData {
+	name: DisplayTheme;
+	cssClass: string;
+}
+
+@Injectable({
+	providedIn: 'root',
+})
+export class DisplayThemeService {
+	private _displayTheme: BehaviorSubject<DisplayTheme>;
+	readonly displayTheme$: Observable<DisplayTheme>;
```

Comment:
azt beszéltük, hogy ahol csak lehet a modern signalokkal fogunk dolgozni, szerintem sokkal egyszerűbb, szebb, és gyakorlásnak is jobb lenne signalt használni a themere, kb ez a tökéletes use case rá

theme = signal<DisplayTheme>('light');
```suggestion
	readonly theme = signal<DisplayTheme>('alapértelmezett');
```